src/sdl/video.c

```diff
@@ -536,12 +533,12 @@ void Vid_SetVideoPalette(int palette)
 
 const char* Vid_GetVideoPaletteName(int palette)
 {
-        return VideoPalettes[palette].name;
+		return VideoPalettes[palette].name;
```

Comment:
This is too much indentation. Side note, can you try to avoid including unrelated changes?